dev/openchpl_data-model.sql

```diff
@@ -3419,6 +3419,20 @@ CREATE TABLE openchpl.pending_certified_product_measure_criteria (
       ON UPDATE CASCADE ON DELETE RESTRICT
 );
 
+CREATE TABLE openchpl.listing_validation_report (
+  id                        bigserial NOT NULL,
+  chpl_product_number       text NOT NULL,
+  product_name              text NOT NULL,
+  certification_status_name text NOT NULL,
+  error_message             text NOT NULL,
+  report_date               timestamp with time zone NOT NULL DEFAULT NOW(),
```

Comment:
Do we have a standard for how we do timestamps? I don't know if we do, or should, or I just don't remember, but didn't we just do a mass conversion of some timestamps to "timestamp without time zone". Would it make sense to use that here? Or do I have it backwards?